src/System.Numerics.Vectors/src/System/Numerics/Register.tt

```diff
@@ -5,7 +5,7 @@
 <#@ import namespace="System" #>
 <#@ import namespace="System.Linq" #>
 <#@ import namespace="System.Runtime.InteropServices" #>
-<#@ import namespace="System.Diagnostics.Contracts" #>
```

Comment:
Yes, you're right about that; I didn't look closely enough here. It's part of the template code so it won't have an effect on any of the generated code.